backend/apps/github/migrations/0021_user_contributions_count.py

```
@@ -0,0 +1,18 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('github', '0020_repositorycontributor_user_contrib_idx'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='user',
```

Ruff (0.8.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)